test/Microsoft.AspNet.Mvc.TagHelpers.Test/SelectTagHelperTest.cs

```diff
@@ -75,14 +75,10 @@ public class SelectTagHelperTest
                     { models, typeof(NestedModel), () => models[0].NestedModel.Text,
                         new NameAndId("[0].NestedModel.Text", "z0__NestedModel_Text"), noneSelected },
 
-                    // TODO: https://github.com/aspnet/Mvc/issues/1468
-                    // Skip last two test cases because DefaultHtmlGenerator evaluates expression name against
-                    // ViewData, not using ModelMetadata.Model. ViewData.Eval() handles simple property paths and some
-                    // dictionary lookups, but not indexing into an array or list. See #1468...
-                    ////{ models, typeof(Model), () => models[1].Text,
-                    ////    new NameAndId("[1].Text", "z1__Text"), outerSelected },
-                    ////{ models, typeof(NestedModel), () => models[1].NestedModel.Text,
-                    ////    new NameAndId("[1].NestedModel.Text", "z1__NestedModel_Text"), innerSelected },
+                    { models, typeof(Model), () => models[1].Text,
+                        new NameAndId("[1].Text", "z1__Text"), outerSelected },
+                    { models, typeof(NestedModel), () => models[1].NestedModel.Text,
+                        new NameAndId("[1].NestedModel.Text", "z1__NestedModel_Text"), innerSelected },
```

Comment:
why?  those entries are `null` which means tests can't tell whether the expression was evaluated correctly